### ISSUE TITLE
begin() doesn't return status

### ISSUE THREAD

**Issue by mikekov** [adafruit/Adafruit_TSL2591_Library#38]
Hi,

Adafruit_TSL2591::begin() always returns true instead of returning real status from a function it calls. Looks like a typo. It breaks a few things down the road that rely on begin().

**Comment by YordanGanev** [comment]
Yea, seems like typo. I was wondering why it shows sensor present when my board had nothing wired on it.  
If you are expecting a status from Adafruit_TSL2591::begin(uint8_t addr) the code below should do the job.

```
/**************************************************************************/
/*!
    @brief   Setups the I2C interface and hardware, identifies if chip is found
    @param   addr The I2C adress of the sensor (Default 0x29)
    @returns True if a TSL2591 is found, false on any failure
*/
/**************************************************************************/
boolean Adafruit_TSL2591::begin(uint8_t addr) {
  return begin(&Wire, addr);
}
```
Greetings!

**Comment by ladyada** [comment]
sure, please submit a PR!

**Comment by caternuson** [comment]
Closing. Fixed with #43.
